<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainGUI</class>
 <widget class="QMainWindow" name="MainGUI">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1145</width>
    <height>819</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>TM MARM-99 YER ISTASYONU</string>
  </property>
  <property name="windowIcon">
   <iconset theme="tm-marm99">
    <normaloff>:/tm-marm99/tm-marm-99-logo.png</normaloff>:/tm-marm99/tm-marm-99-logo.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">QMainWindow{
background-image: url(:/tm_marm99/space.jpg);
}


QStatusBar{
background-color: qlineargradient(spread:pad, x1:1, y1:1, x2:1, y2:0, stop:0.288557 rgba(0, 0, 0, 224), stop:1 rgba(235, 235, 235, 0));
}
</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="styleSheet">
    <string notr="true">margin:0px;
padding:0px;
background:rgba(0,0,0,0.7);
border-width:3px;
</string>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <property name="spacing">
     <number>7</number>
    </property>
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <layout class="QHBoxLayout" name="title_bar_layout">
      <property name="spacing">
       <number>0</number>
      </property>
      <item>
       <widget class="QFrame" name="left_bar">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="minimumSize">
         <size>
          <width>100</width>
          <height>38</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>38</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">QFrame{
background:rgba(0,0,0,0.38);
}
QFrame{
border-radius:none;
}

QFrame:hover{
background:rgba(0,0,0,0.40);
}</string>
        </property>
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <property name="spacing">
          <number>0</number>
         </property>
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QLabel" name="label">
           <property name="maximumSize">
            <size>
             <width>35</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">QLabel{
background:#3b3937;
color:#fff;
border-radius:none;
padding-top:-2px;
padding-left:7px;
}
</string>
           </property>
           <property name="text">
            <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;img src=&quot;:/tm_marm99/tm-marm-99-logo-bw.png&quot; width=&quot;20&quot; height=&quot;20&quot;/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="title_label">
           <property name="font">
            <font>
             <pointsize>-1</pointsize>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">QLabel{
background:#3b3937;
color:rgb(227, 227, 227);
border-radius:none;
padding-top:-1px;
padding-left:4px;

font-size:15px;
}
</string>
           </property>
           <property name="text">
            <string>TM MARM-99 YER ISTASYONU</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QFrame" name="right_bar">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="minimumSize">
         <size>
          <width>180</width>
          <height>38</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>180</width>
          <height>38</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">QFrame{
background:rgba(0,0,0,0.38);
}
QFrame{
border-radius:none;
}

QFrame:hover{
background:rgba(0,0,0,0.40);
}</string>
        </property>
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <property name="spacing">
          <number>0</number>
         </property>
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QPushButton" name="button_minimize">
           <property name="minimumSize">
            <size>
             <width>60</width>
             <height>38</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>60</width>
             <height>38</height>
            </size>
           </property>
           <property name="font">
            <font>
             <pointsize>13</pointsize>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">QPushButton{
background:#3b3937;
color:#fff;

border-radius:none;

padding-top:-7px
}


QPushButton:hover{
background:rgb(79, 76, 74);
color:white
}</string>
           </property>
           <property name="text">
            <string>_</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="button_max_min">
           <property name="minimumSize">
            <size>
             <width>60</width>
             <height>38</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>60</width>
             <height>38</height>
            </size>
           </property>
           <property name="font">
            <font>
             <pointsize>11</pointsize>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">QPushButton{
background:#3b3937;
color:#fff;

border-radius:none;
padding-top:-3px
}


QPushButton:hover{
background:#4f4c4a;
color:white
}</string>
           </property>
           <property name="text">
            <string>ðŸ—–</string>
           </property>
          </widget>
         </item>
         <item alignment="Qt::AlignLeft">
          <widget class="QPushButton" name="button_close">
           <property name="minimumSize">
            <size>
             <width>60</width>
             <height>38</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>60</width>
             <height>38</height>
            </size>
           </property>
           <property name="font">
            <font>
             <pointsize>11</pointsize>
            </font>
           </property>
           <property name="cursor">
            <cursorShape>ArrowCursor</cursorShape>
           </property>
           <property name="styleSheet">
            <string notr="true">QPushButton{
background:#3b3937;
color:#fff;

border-radius:none;
}


QPushButton:hover{
background:#e81123;
color:white
}</string>
           </property>
           <property name="text">
            <string>âœ•</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QGridLayout" name="main_layout">
      <item row="0" column="0">
       <layout class="QHBoxLayout" name="top_layout">
        <item>
         <layout class="QVBoxLayout" name="port"/>
        </item>
        <item>
         <layout class="QVBoxLayout" name="data">
          <item>
           <widget class="QPushButton" name="pushButton">
            <property name="text">
             <string>PushButton</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="data_transfer"/>
        </item>
       </layout>
      </item>
      <item row="1" column="0">
       <layout class="QHBoxLayout" name="mid_layout"/>
      </item>
      <item row="2" column="0">
       <layout class="QHBoxLayout" name="bottom_layout"/>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar">
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>35</height>
    </size>
   </property>
   <property name="sizeGripEnabled">
    <bool>true</bool>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="../images/tm_marm99.qrc"/>
 </resources>
 <connections/>
</ui>
